.github/workflows/std-workflows.yml

```diff
@@ -0,0 +1,34 @@
+name: GitHub Actions CI/CD
+run-name: Deploying Local CI.
+on:
+  push:
+    branches:
+      - exercise-04
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "The job was automatically triggered by a Push event."
+     
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Docker Login
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+
+      - name: Building flask application.
+        run: |
+          make flaskapp_build
+
+      - name: Run docker compose.
+        run: |
+          make docker_build
+
+      - name: Push the Image into dockerHub.
+        run: |
+          make docker_push
+        env:
+          REPO_NAME: aashirathore111
```

Comment:
your understanding of Docker REPOSITORY is incorrect.

Please learn about the difference between a Docker Registry, Docker Username and Docker Repository.

Reply:
I read some of the points I mentioned in my Trello card for this.
To resolve this I pass the username parameter to makefile.